demos/speech_recognition_wav2vec_demo/python/speech_recognition_wav2vec_demo.py

```diff
@@ -51,23 +53,25 @@ class Wav2Vec:
     words_delimiter = '|'
     pad_token = '<pad>'
 
-    def __init__(self, core, model_path, input_shape, device, vocab_file):
+    def __init__(self, core, model_path, input_shape, device, vocab_file, dynamic_flag):
         log.info('Reading model {}'.format(model_path))
         model = core.read_model(model_path)
         if len(model.inputs) != 1:
             raise RuntimeError('Wav2Vec must have one input')
         self.input_tensor_name = model.inputs[0].get_any_name()
-        model_input_shape = model.inputs[0].shape
+        self.dynamic = model.is_dynamic() and dynamic_flag
```

Comment:
I think user flag should have priority under model shape, if model is static, but dynamic flag you can specify reshape with dynamic shape (-1, -1)